src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Quadrangle {
+    public IsoscelesTrapezoid(String color, int width, int height) {
+        super(color, width, height);
+    }
+
+    @Override
+    public double area() {
+        return super.area();
+    }
+
+    @Override
+    public void draw() {
+        System.out.print("Figure: isoscelesTrapezoid");
+        super.draw();
+    }
```

Comment:
Don't use abstract classes to set behavior for classes. It seems like the 'Quadrangle' class is being used to provide a default implementation of methods, which may not be appropriate if those methods are conceptually different or if 'Quadrangle' is meant to be a generic concept without a specific implementation. Consider using interfaces and composition instead.